resident/resident-service/src/main/java/io/mosip/resident/validator/RequestValidator.java

```diff
@@ -596,10 +596,10 @@ public void validateAuthType(List<String> authType, String msg) {
 			audit.setAuditRequestDto(EventEnum.getEventEnumWithValue(EventEnum.INPUT_INVALID, "authTypes", msg));
 			throw new InvalidInputException("authTypes");
 		}
-		String[] authTypesArray = authTypes.split(",");
+		String[] authTypesArray = authTypes.toLowerCase().split(",");
 		List<String> authTypesAllowed = new ArrayList<>(Arrays.asList(authTypesArray));
 		for (String type : authType) {
```

Comment:
Please rename `authType` to `authTypesList`

Reply:
ok